avvylib/src/main/java/xyz/schwaab/avvylib/AvatarView.kt

```diff
@@ -369,6 +413,7 @@ class AvatarView : ImageView {
 
         borderRect.set(calculateBounds())
         borderRadius = min((borderRect.height() - currentBorderThickness) / 2.0f, (borderRect.width() - currentBorderThickness) / 2.0f)
+        badgeRadius = borderRadius / 4
```

Comment:
This makes it impossible to have a very thin border with a badge.

I think that the `borderRadius` should be customisable just like `badgeColor`, `badgeStrokeColor` and `badgeStrokeWidth`. Maybe making it `8dp` or similar by default, if not specified.

Reply:
Thanks for the review, @vitorhugods.

I've made `badgeRadius` customizable. Default value set to `8dp`.
I've also added `badgePosition` attribute to position the badge in the four corners of the `borderRect`. Default value set to `BOTTOM_RIGHT`.